test/evergreen.yml

```diff
@@ -1593,6 +1593,65 @@ tasks:
           display_name: Coverage report
           remote_file: wiredtiger/${build_variant}/${revision}/coverage_report/coverage_report_${build_id}.html
 
+  # creating bunch of spinlock tests so they can be scheduled in parallel
+  - name: test-spinlock-gcc-1
+    tags: ["test-spinlock"]
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
```

Comment:
I'd prefer to have a separate compile task for this though I know we need to recompile this anyway. But I'm OK with this for now, up to @lukech

Reply:
I actually did have a seperate task but it simply looked unintuitive to compile code again after `fetch artifacts`. Also it saves the cost of uploading/downloading aritifacts between tasks (not sure if that actually helps with AWS billing :) ).